### ISSUE TITLE
508-defect-1 ❗ [SCREENREADER]: Error messages specific to each sections are not associated with their respective accordions

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va-forms-system-core#474]
@obliviga , following up on [testing the representative documentation](https://github.com/department-of-veterans-affairs/va-forms-system-core/issues/276), I noticed the following issues:

- When there are issues in separate, collapsed accordions, the "We're sorry" message is announced
- When navigating to each of the separate, collapsed accordions, it's not clear as to which accordion contains issues to be corrected because their respective error messages...
  - `role="alert"` is cancelled out by the first "we're sorry" message (which is OK, we probably don't want all of them announcing at once)
  - aren't programmatically linked to the accordion their describing as containing the error

https://user-images.githubusercontent.com/14154792/179545866-bef14789-73a2-44f6-922c-6599f663f89e.mov

To fix this particular edge case, we may want to consider using `aria-describedby` to associate each of the error messages to the buttons that expand and collapse the fields as [documented in the bandaid solution](https://github.com/department-of-veterans-affairs/va-forms-system-core/issues/276) of the original ticket.

**Comment by obliviga** [comment]
The branch has been successfully deployed: ![image.png](https://images.zenhubusercontent.com/6283eaffdea4c21364d67e5a/0600aff6-2635-43be-8d3c-093a21bef47a)

http://cb0d7c38d30c6ea57a41877d0b0a986c.review.vetsgov-internal/health-care/apply/application/review-and-submit
